src/main/java/sg4e/ff4stats/fe/FlagSet.java

```diff
@@ -200,14 +207,27 @@ public static FlagSet fromString(String text) {
         
         for(String part : parts) {
             Flag previousFlag = null;
-            while(part.length() > 0) {
-                Flag flag = FlagVersion.getFlagFromFlagString(version, part, previousFlag);
+            while(part.length() > 0) {                
+                System.out.println("-----\nOriginal part: " + part);
```

Comment:
Use call to `LOG.debug` or `LOG.info` instead of printing to stdout.

Example of instantiating the logger: https://github.com/sg4e/FF4StatsLib/blob/113284efd0281499b1f6aec1bec4f7a6e0108c4c/src/main/java/sg4e/ff4stats/party/Armor.java#L44